src/styles/main.scss

```diff
@@ -1,3 +1,33 @@
+@import "./utils/variables";
+@import "header";
+@import "./blocks/stars";
+@import "./blocks/card";
+
+:root {
+  font-family: "Roboto", "Montserrat", "Albert Sans", sans-serif;
+}
```

Comment:
it's good to add css variables to `:root` selector but not a font. move the style to `html` or `body` slector